chrome/content/zotero/itemTree.jsx

```diff
@@ -1899,8 +1925,9 @@ var ItemTree = class ItemTree extends LibraryTree {
 	};
 	
 	isContainer = (index) => {
-		return this.getRow(index).ref.isRegularItem();
-	};
+		let item = this.getRow(index).ref;
+		return item.isRegularItem() || (item.isFileAttachment() && item.numAnnotations());
```

Comment:
I don't think there's a reason to have both `isContainer()` and `isContainerEmpty()`, but since we do, technically the condition is `item.isRegularItem() || item.isFileAttachment()`. The `item.numAnnotations()` should be handled in `isContainerEmpty()`.

Reply:
Great catch, thank you! In fact, it already is handled in `isContainerEmpty`, so this must have been just a missed leftover from my initial attempt of getting annotations to show up in itemTree